challenge.md

```diff
@@ -5,19 +5,24 @@
 Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+git clone "SSH clone URL"
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
-
+git diff
```

Comment:
We were looking for `git status`, but given the wording of the question, I can see why you might have written this. No points off.